mne/time_frequency/tfr.py

```diff
@@ -237,16 +240,16 @@ def _cwt(X, Ws, mode="same", decim=1, use_fft=True):
                 this_slice = slice(offset // decim, (offset + sz) // decim)
                 if use_fft:
                     ret = _centered(ret, sz)
-                tfr[ii, this_slice] = ret[::decim]
+                tfr[ii, this_slice] = ret[tmin:tmax:decim]
             else:
                 if use_fft:
                     ret = _centered(ret, n_times)
-                tfr[ii, :] = ret[::decim]
+                tfr[ii, :] = ret[tmin:tmax:decim]
         yield tfr
 
 
 def cwt_morlet(X, sfreq, freqs, use_fft=True, n_cycles=7.0, zero_mean=False,
-               decim=1):
+               decim=1, time_range=(0, None)):
```

Comment:
In the baseline stuff, 0 and None mean other things right? 

E.g. (None, 0) means from the beginning to the zero point. It would probably better to have the same meaning here.